nextcord/flags.py

```diff
@@ -486,6 +486,14 @@ def known_spammer(self):
         """
         return UserFlags.known_spammer.value
 
+    @flag_value
+    def active_developer(self):
+        """:class:`bool`: Returns ``True`` if the user is a Active Developer.
```

Comment:
```suggestion
        """:class:`bool`: Returns ``True`` if the user is an Active Developer.
```

Reply:
Looks like this happened because I copied the documentation too early (when it still wasn't merged) 😅 Fixed in https://github.com/nextcord/nextcord/pull/897/commits/e1e129ac5fc1ab3cea8d43f5eed9eac7e01b591b